nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -476,7 +475,8 @@ in
         default = false;
         type = types.bool;
         description = lib.mdDoc ''
-          Enable recommended brotli settings. Learn more about compression in Brotli format [here](https://github.com/google/ngx_brotli/blob/master/README.md).
+          Enable recommended brotli settings.
+          Learn more about compression in Brotli format [here](https://github.com/google/ngx_brotli/blob/master/README.md).
```

Comment:
```suggestion
          Learn more about compression in Brotli format [here](https://github.com/google/ngx_brotli/).
```
We once link the readme directly and once the repo, just do the same in both links

Reply:
Fixed.